ephios/extra/middleware.py

```diff
@@ -36,3 +39,17 @@ def __call__(self, request):
             except Notification.DoesNotExist:
                 pass
         return response
+
+
+class EphiosMediaFileMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        response = self.get_response(request)
+        if (
+            request.get_host() == urlsplit(settings.GET_USERCONTENT_URL()).netloc
+            and request.resolver_match.url_name != "document"
+        ):
+            return redirect(urljoin(settings.GET_SITE_URL(), request.path))
+        return response
```

Comment:
yeah good point. We could move the list of "media views" to the settings

Reply:
I still don't understand what it does

Reply:
if you request the usercontent domain, but on a view that does not serve usercontent (e.g. ephiosusercontent.de/accounts/login), it will redirect you to the same view but on the regular domain. This prevents the app from being served under the content domain, and therefore on the same domain as the usercontent